apache-hbase-mixin/alerts/alerts.libsonnet

```diff
@@ -0,0 +1,113 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'apache-hbase-alerts',
+        rules: [
+          {
+            alert: 'HighHeapMemUsage',
+            expr: |||
+              100 * sum without(context, hostname, processname) (jvm_metrics_mem_heap_used_m / clamp_min(jvm_metrics_mem_heap_committed_m, 1))  > %(alertsHighHeapMemUsage)s
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              summary: 'There is a limited amount of heap memory available to the JVM.',
+              description:
+                (
+                  'The heap memory usage for the JVM on instance {{$labels.instance}} in cluster {{$labels.hbase_cluster}} is {{printf "%%.0f" $labels.value}} percent, which is above the threshold of %(alertsHighHeapMemUsage)s'
+                ) % $._config,
+            },
+          },
+          {
+            alert: 'HighNonHeapMemUsage',
+            expr: |||
+              100 * sum without(context, hostname, processname) (jvm_metrics_mem_non_heap_used_m / clamp_min(jvm_metrics_mem_non_heap_committed_m, 1))  > %(alertsHighNonHeapMemUsage)s
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              summary: 'There is a limited amount of non-heap memory available to the JVM.',
+              description:
+                (
+                  'The non-heap memory usage for the JVM on instance {{$labels.instance}} in cluster {{$labels.hbase_cluster}} is {{printf "%%.0f" $labels.value}} percent, which is above the threshold of %(alertsHighNonHeapMemUsage)s'
+                ) % $._config,
+            },
+          },
+          {
+            alert: 'DeadRegionServer',
+            expr: |||
+              server_num_dead_region_servers > %(alertsDeadRegionServer)s
```

Comment:
this metric has a lot of labels associated with it... not sure if it's worthwhile removing some of them with a sum without or if you're happy with as is? also applies to the other non JVM alerts

Reply:
Yeah there are quite a few, but I don't think any of them will alter the calculation, so they should be fine to keep around in case they provide some specific context to the user that could be useful.